ext_emconf.php

```diff
@@ -11,7 +11,7 @@
     'author_company' => 'Leuchtfeuer Digital Marketing',
     'author_email' => 'dev@Leuchtfeuer.com',
     'clearCacheOnLoad' => 0,
-    'version' => '4.0.1',
+    'version' => '4.1.0',
```

Comment:
This version contains breaking changes. Raise version to 5.0.0 or get rid of these breaking changes.